test/chrono-test.cc

```diff
@@ -544,14 +544,12 @@ TEST(chrono_test, negative_durations) {
 
 TEST(chrono_test, special_durations) {
   EXPECT_EQ(
-      "40.",
-      fmt::format("{:%S}", std::chrono::duration<double>(1e20)).substr(0, 3));
+      "40",
+      fmt::format("{:%S}", std::chrono::duration<double>(1e20)).substr(0, 2));
```

Comment:
According to the spec, the decimal point should be printed only if the duration "period" type is more precise than seconds:

> If the precision of the input cannot be exactly represented with seconds, then the format is a decimal floating-point number with a fixed format and a precision matching that of the precision of the input

 This can be determined statically at compile time and does not depend on runtime values of the duration. The test I modified has seconds precision, so I removed the decimal point.

In my local instance of Visual Studio 2022, the following test passes:

`std::format("{:%S}", std::chrono::duration<double>{1.234}) == std::string("01")`

Reply:
Shall we check that there is no decimal point then?